java/src/main/java/com/powsybl/python/network/NetworkCFunctions.java

```diff
@@ -863,4 +868,43 @@ public static void setMinValidationLevel(IsolateThread thread, ObjectHandle netw
         });
     }
 
+    @CEntryPoint(name = "createBranchFeederBays")
+    public static void createBranchFeederBays(IsolateThread thread, ObjectHandle networkHandle,
+                                        CCharPointer bbsId1,
+                                        CCharPointer bbsId2,
+                                        int positionOrder1,
+                                        int positionOrder2,
+                                        CCharPointer direction1,
+                                        CCharPointer direction2,
+                                        DataframePointer cDataframeLine,
+                                        ExceptionHandlerPointer exceptionHandlerPtr) {
+        doCatch(exceptionHandlerPtr, () -> {
+            Network network = ObjectHandles.getGlobal().get(networkHandle);
+            String bbsId1Str = CTypeUtil.toString(bbsId1);
+            String bbsId2Str = CTypeUtil.toString(bbsId2);
+            String direction1Str = CTypeUtil.toString(direction1);
+            String direction2Str = CTypeUtil.toString(direction2);
+
+            UpdatingDataframe df = createDataframe(cDataframeLine);
+            LineAdder adder = network.newLine();
```

Comment:
We should try to re-use more the code of line creation that already exists, like we're trying to do for injections.